src/error.rs

```diff
@@ -168,6 +175,7 @@ impl Display for ErrorCode {
             ErrorCode::UnexpectedPacket(ref packet_type) => f.write_fmt(format_args!("unexpected packet type {packet_type}")),
             ErrorCode::UnknownCmd => f.write_str("unknown command"),
             ErrorCode::UnknownEvent(ref event) => f.write_fmt(format_args!("unknown event {event}")),
+            ErrorCode::CommandFailed(ref reason) => f.write_fmt(format_args!("command failed - {reason}")),
```

Comment:
I’d like to make this aligned with other strings:

```suggestion
            ErrorCode::CommandFailed(ref reason) => f.write_fmt(format_args!("command failed: {reason}")),
```

Reply:
Done